### ISSUE TITLE
Walkback "a String does not understand #'abs'" when setting breakpoint

### ISSUE THREAD

**Issue by LisaAlmarode** [GemTalk/JadeiteForPharo#141]
With the curor positioned in non-selector code such as self, a temp variable, or a optimized selector, quite frequently the breakpoint menu item says "Add breakpoint at step point * ".  This results in a walkback (below).  Closing that gives me another debugger open with cull sent to nil.  Nothing terrible ensued, though.

```
a MessageNotUnderstood occurred (error 2010), a String does not understand  #'abs'

[1] ProcessorScheduler >> _switchFrom:to: @2 line 8
[2] ProcessorScheduler >> _reschedule @7 line 7
[3] ProcessorScheduler >> _suspendProcess: @8 line 8
[4] GsProcess >> suspend @7 line 8
[5] [] in RowanService >> setDebugActionBlock @45 line 43
[6] ExecBlock1 (ExecBlock) >> cull: @7 line 7
[7] MessageNotUnderstood >> defaultAction @7 line 5
[8] MessageNotUnderstood (AbstractException) >> _defaultAction @4 line 4
[9] MessageNotUnderstood (AbstractException) >> _signal @2 line 20
[10] MessageNotUnderstood (AbstractException) >> signal @2 line 47
[11] String (Object) >> doesNotUnderstand: @9 line 10
[12] String (Object) >> _doesNotUnderstand:args:envId:reason: @12 line 17
[13] GsNMethod >> _breakOperation:forStepPoint:breakpointLevel: @2 line 9
[14] GsNMethod >> setBreakAtStepPoint:breakpointLevel: @3 line 12
[15] RowanMethodService >> setBreakAt: @8 line 6
[16] RowanMethodService (Object) >> perform:withArguments: @1 line 12
[17] RowanMethodService (RowanService) >> servicePerform:withArguments:shouldUpdate: @5 line 7
[18] RowanMethodService >> servicePerform:withArguments: @9 line 7
[19] [] in RowanService >> executeCommand @21 line 11
[20] ExecBlock0 (ExecBlock) >> on:do: @3 line 44
[21] [] in RowanService >> executeCommand @10 line 12
[22] ExecBlock0 (ExecBlock) >> on:do: @3 line 44
[23] RowanMethodService (RowanService) >> executeCommand @4 line 19
[24] RowanMethodService (Object) >> perform:withArguments: @1 line 12
[25] RsrMessageSend >> perform @5 line 4
[26] [] in RsrSendMessage >> perform:answerUsing: @9 line 6
[27] ExecBlock0 (ExecBlock) >> on:do: @3 line 44
[28] RsrSendMessage >> perform:answerUsing: @3 line 9
[29] [] in RsrSendMessage >> executeFor: @34 line 21
[30] ExecBlock0 (ExecBlock) >> on:do: @3 line 44
[31] [] in RsrSendMessage >> executeFor: @9 line 22
[32] ExecBlock0 (ExecBlock) >> ensure: @2 line 12
[33] RsrSendMessage >> executeFor: @4 line 26
[34] [] in RsrConnection >> _receivedCommand: @10 line 7
[35] [] in RsrProcessModel >> fork:named: @7 line 5
[36] ExecBlock >> valueWithArguments: @1 line 7
[37] GsProcess >> _start @7 line 16
[38] GsNMethod class >> _gsReturnToC @1 line 11
===============================================
[1] ProcessorScheduler >> _switchFrom:to: @2 line 8
	receiver oop(23205633) == aProcessorScheduler

instance variables:
	-activeProcess oop(45191169) == GsProcess(oop=45191169, status=active, priority=15, block=anExecBlock, args=anArray( ), 
   GsProcess >> _reportAt: @7 line 10
   GsProcess >> _reportOfSize: @11 line 11
   GsProcess >> _reportString ...
	-criticalMethods oop(45188865) == anIdentitySet( GsNMethod AbstractException>>_signalTimeout:, GsNMethod AbstractException>>_signalAsync:, GsNMethod GsProcess class>>_forkBlock:with:env:prio:, GsNMethod GsProcess class>>_scheduler, Gs...
	-delayQueue oop(19034881) == aSortedCollection( )
	-lastGroup oop(2) == 0
	-pollResults oop(19037185) == anArray( )
	-readyQueue oop(19049217) == aSortedCollection( )
	-socketsPolled oop(18973953) == anArray( aGsSignalingSocket, aGsSignalingSocket)
	-suspendedSet oop(18975745) == anIdentitySet( GsProcess(oop=26702849, status=suspended, priority=15, block=anExecBlock, args=anArray( ), 
   ProcessorScheduler >> _switchFrom:to: @2 line 8
   ProcessorScheduler >> _reschedule @7 li...
	-timeoutThreadActive oop(12) == false
	-waitingSet oop(18974977) == anIdentitySet( GsProcess(oop=18366721, status=waiting on a Semaphore, priority=30, block=anExecBlock, args=anArray( ), 
   ProcessorScheduler >> _switchFrom:to: @2 line 8
   ProcessorScheduler >> _res...

temporary variables and parameters:
	newProcess oop(45191169) == GsProcess(oop=45191169, status=active, priority=15, block=anExecBlock, args=anArray( ), 
   GsProcess >> _reportAt: @7 line 10
   GsProcess >> _reportOfSize: @11 line 11
   GsProcess >> _reportString ...
	oldProcess oop(45212161) == GsProcess(oop=45212161, status=suspended, priority=15, block=anExecBlock, args=anArray( ), 
   ProcessorScheduler >> _switchFrom:to: @2 line 8
   ProcessorScheduler >> _reschedule @7 line 7
   Process...
----------------------------------------------------
[2] ProcessorScheduler >> _reschedule @7 line 7
	receiver oop(23205633) == aProcessorScheduler

instance variables:
	-activeProcess oop(45191169) == GsProcess(oop=45191169, status=active, priority=15, block=anExecBlock, args=anArray( ), 
   GsProcess >> _reportAt: @7 line 10
   GsProcess >> _reportOfSize: @11 line 11
   GsProcess >> _reportString ...
	-criticalMethods oop(45188865) == anIdentitySet( GsNMethod AbstractException>>_signalTimeout:, GsNMethod AbstractException>>_signalAsync:, GsNMethod GsProcess class>>_forkBlock:with:env:prio:, GsNMethod GsProcess class>>_scheduler, Gs...
	-delayQueue oop(19034881) == aSortedCollection( )
	-lastGroup oop(2) == 0
	-pollResults oop(19037185) == anArray( )
	-readyQueue oop(19049217) == aSortedCollection( )
	-socketsPolled oop(18973953) == anArray( aGsSignalingSocket, aGsSignalingSocket)
	-suspendedSet oop(18975745) == anIdentitySet( GsProcess(oop=26702849, status=suspended, priority=15, block=anExecBlock, args=anArray( ), 
   ProcessorScheduler >> _switchFrom:to: @2 line 8
   ProcessorScheduler >> _reschedule @7 li...
	-timeoutThreadActive oop(12) == false
	-waitingSet oop(18974977) == anIdentitySet( GsProcess(oop=18366721, status=waiting on a Semaphore, priority=30, block=anExecBlock, args=anArray( ), 
   ProcessorScheduler >> _switchFrom:to: @2 line 8
   ProcessorScheduler >> _res...

temporary variables and parameters:
	newProc oop(45191169) == GsProcess(oop=45191169, status=active, priority=15, block=anExecBlock, args=anArray( ), 
   GsProcess >> _reportAt: @7 line 10
   GsProcess >> _reportOfSize: @11 line 11
   GsProcess >> _reportString ...
	oldProc oop(45212161) == GsProcess(oop=45212161, status=suspended, priority=15, block=anExecBlock, args=anArray( ), 
   ProcessorScheduler >> _switchFrom:to: @2 line 8
   ProcessorScheduler >> _reschedule @7 line 7
   Process...
----------------------------------------------------
[3] ProcessorScheduler >> _suspendProcess: @8 line 8
	receiver oop(23205633) == aProcessorScheduler

instance variables:
	-activeProcess oop(45191169) == GsProcess(oop=45191169, status=active, priority=15, block=anExecBlock, args=anArray( ), 
   GsProcess >> _reportAt: @7 line 10
   GsProcess >> _reportOfSize: @11 line 11
   GsProcess >> _reportString ...
	-criticalMethods oop(45188865) == anIdentitySet( GsNMethod AbstractException>>_signalTimeout:, GsNMethod AbstractException>>_signalAsync:, GsNMethod GsProcess class>>_forkBlock:with:env:prio:, GsNMethod GsProcess class>>_scheduler, Gs...
	-delayQueue oop(19034881) == aSortedCollection( )
	-lastGroup oop(2) == 0
	-pollResults oop(19037185) == anArray( )
	-readyQueue oop(19049217) == aSortedCollection( )
	-socketsPolled oop(18973953) == anArray( aGsSignalingSocket, aGsSignalingSocket)
	-suspendedSet oop(18975745) == anIdentitySet( GsProcess(oop=26702849, status=suspended, priority=15, block=anExecBlock, args=anArray( ), 
   ProcessorScheduler >> _switchFrom:to: @2 line 8
   ProcessorScheduler >> _reschedule @7 li...
	-timeoutThreadActive oop(12) == false
	-waitingSet oop(18974977) == anIdentitySet( GsProcess(oop=18366721, status=waiting on a Semaphore, priority=30, block=anExecBlock, args=anArray( ), 
   ProcessorScheduler >> _switchFrom:to: @2 line 8
   ProcessorScheduler >> _res...

temporary variables and parameters:
	aProcess oop(45212161) == GsProcess(oop=45212161, status=suspended, priority=15, block=anExecBlock, args=anArray( ), 
   ProcessorScheduler >> _switchFrom:to: @2 line 8
   ProcessorScheduler >> _reschedule @7 line 7
   Process...
	onQ oop(20) == nil
	suspSet oop(18975745) == anIdentitySet( GsProcess(oop=26702849, status=suspended, priority=15, block=anExecBlock, args=anArray( ), 
   ProcessorScheduler >> _switchFrom:to: @2 line 8
   ProcessorScheduler >> _reschedule @7 li...
----------------------------------------------------
[4] GsProcess >> suspend @7 line 8
	receiver oop(45212161) == GsProcess(oop=45212161, status=suspended, priority=15, block=anExecBlock, args=anArray( ), 
   ProcessorScheduler >> _switchFrom:to: @2 line 8
   ProcessorScheduler >> _reschedule @7 line 7
   Process...

instance variables:
	-arStack oop(45190401) == aGsStackBuffer( 64, GsNMethod ProcessorScheduler>>_switchFrom:to:, -1888, 136, GsProcess(oop=45191169, status=active, priority=15, block=anExecBlock, args=anArray( ), 
   GsProcess >> _reportAt: @7 li...
	-args oop(233217) == anArray( )
	-block oop(45190657) == anExecBlock
	-blockResult oop(276) == remoteNil
	-breakpointLevel oop(10) == 1
	-clientData oop(20) == nil
	-dbgFpsCache oop(45190913) == anArray( 3, 11, 17, 22, 29, 34, 41, 46, 51, 55, 62, 67, 77, 84, 91, 96, 102, 110, 117, 124, 131, 138, 147, 151, 157, 163, 170, 177, 184, 191, 198, 206, 214, 220, 225, 229, 236, 244)
	-debugActionBlock oop(45191425) == anExecBlock1
	-environment oop(20) == nil
	-ffiErrno oop(2) == 0
	-group oop(20) == nil
	-interruptFlag_stEntryCount oop(10) == 1
	-ioSelectResult oop(20) == nil
	-joiners oop(20) == nil
	-lastBreakpt oop(20) == nil
	-modeInfo oop(86033565698) == 10754195712
	-msgEnvironmentId oop(2) == 0
	-name oop(45191681) == 'RowanMethodService>>executeCommand'
	-onQueue oop(18975745) == anIdentitySet( GsProcess(oop=26702849, status=suspended, priority=15, block=anExecBlock, args=anArray( ), 
   ProcessorScheduler >> _switchFrom:to: @2 line 8
   ProcessorScheduler >> _reschedule @7 li...
	-parentProcess oop(18366977) == GsProcess(oop=18366977, status=waiting on a GsSignalingSocket, priority=30, block=anExecBlock, args=anArray( ), 
   ProcessorScheduler >> _switchFrom:to: @2 line 8
   ProcessorScheduler >> _reschedule...
	-priority_use oop(131941395333162) == 16492674416645
	-signalTime oop(20) == nil
	-signalledException oop(20) == nil
	-stackId oop(18446744073709551610) == -1
	-stackSerialNum oop(62013669418) == 7751708677
	-startSeconds oop(11269708529815813222) == 750880345.2326591
	-topFpOffset oop(18446744073709536002) == -1952
	-uaCount_depth oop(306) == 38
	-waitingOn oop(20) == nil

temporary variables and parameters:
----------------------------------------------------
[5] [] in RowanService >> setDebugActionBlock @45 line 43
	receiver oop(45191425) == anExecBlock1

instance variables:

temporary variables and parameters:
	compileErrorService oop(20) == nil
	debuggerResult oop(20) == nil
	debuggerResult oop(20) == nil
	ex oop(45192193) == a MessageNotUnderstood occurred (error 2010), a String does not understand  #'abs'
	proc oop(45212161) == GsProcess(oop=45212161, status=suspended, priority=15, block=anExecBlock, args=anArray( ), 
   ProcessorScheduler >> _switchFrom:to: @2 line 8
   ProcessorScheduler >> _reschedule @7 line 7
   Process...
	self oop(49702657) == aRowanMethodService(OrderPricer>>numOfRegularDefaultPricedPlants)
	suspendedProcess oop(45212161) == GsProcess(oop=45212161, status=suspended, priority=15, block=anExecBlock, args=anArray( ), 
   ProcessorScheduler >> _switchFrom:to: @2 line 8
   ProcessorScheduler >> _reschedule @7 line 7
   Process...
	suspendedProcess oop(45212161) == GsProcess(oop=45212161, status=suspended, priority=15, block=anExecBlock, args=anArray( ), 
   ProcessorScheduler >> _switchFrom:to: @2 line 8
   ProcessorScheduler >> _reschedule @7 line 7
   Process...
----------------------------------------------------
[6] ExecBlock1 (ExecBlock) >> cull: @7 line 7
	receiver oop(45191425) == anExecBlock1

instance variables:

temporary variables and parameters:
	anArg oop(45192193) == a MessageNotUnderstood occurred (error 2010), a String does not understand  #'abs'
----------------------------------------------------
[7] MessageNotUnderstood >> defaultAction @7 line 5
	receiver oop(45192193) == a MessageNotUnderstood occurred (error 2010), a String does not understand  #'abs'

instance variables:
	-currGsHandler oop(20) == nil
	-envId oop(2) == 0
	-gsArgs oop(45188609) == anArray( )
	-gsDetails oop(20) == nil
	-gsNumber oop(16082) == 2010
	-gsReason oop(20) == nil
	-gsResumable oop(268) == true
	-gsStack oop(20) == nil
	-gsTrappable oop(268) == true
	-messageText oop(45189889) == 'a MessageNotUnderstood occurred (error 2010), a String does not understand  #''abs'''
	-receiver oop(45211393) == '*'
	-selector oop(5408769) == #'abs'
	-tag oop(20) == nil

temporary variables and parameters:
	blk oop(45191425) == anExecBlock1
	res oop(20) == nil
----------------------------------------------------
[8] MessageNotUnderstood (AbstractException) >> _defaultAction @4 line 4
	receiver oop(45192193) == a MessageNotUnderstood occurred (error 2010), a String does not understand  #'abs'

instance variables:
	-currGsHandler oop(20) == nil
	-envId oop(2) == 0
	-gsArgs oop(45188609) == anArray( )
	-gsDetails oop(20) == nil
	-gsNumber oop(16082) == 2010
	-gsReason oop(20) == nil
	-gsResumable oop(268) == true
	-gsStack oop(20) == nil
	-gsTrappable oop(268) == true
	-messageText oop(45189889) == 'a MessageNotUnderstood occurred (error 2010), a String does not understand  #''abs'''
	-receiver oop(45211393) == '*'
	-selector oop(5408769) == #'abs'
	-tag oop(20) == nil

temporary variables and parameters:
	res oop(20) == nil
----------------------------------------------------
[9] MessageNotUnderstood (AbstractException) >> _signal @2 line 20
	receiver oop(45192193) == a MessageNotUnderstood occurred (error 2010), a String does not understand  #'abs'

instance variables:
	-currGsHandler oop(20) == nil
	-envId oop(2) == 0
	-gsArgs oop(45188609) == anArray( )
	-gsDetails oop(20) == nil
	-gsNumber oop(16082) == 2010
	-gsReason oop(20) == nil
	-gsResumable oop(268) == true
	-gsStack oop(20) == nil
	-gsTrappable oop(268) == true
	-messageText oop(45189889) == 'a MessageNotUnderstood occurred (error 2010), a String does not understand  #''abs'''
	-receiver oop(45211393) == '*'
	-selector oop(5408769) == #'abs'
	-tag oop(20) == nil

temporary variables and parameters:
	res oop(20) == nil
----------------------------------------------------
[10] MessageNotUnderstood (AbstractException) >> signal @2 line 47
	receiver oop(45192193) == a MessageNotUnderstood occurred (error 2010), a String does not understand  #'abs'

instance variables:
	-currGsHandler oop(20) == nil
	-envId oop(2) == 0
	-gsArgs oop(45188609) == anArray( )
	-gsDetails oop(20) == nil
	-gsNumber oop(16082) == 2010
	-gsReason oop(20) == nil
	-gsResumable oop(268) == true
	-gsStack oop(20) == nil
	-gsTrappable oop(268) == true
	-messageText oop(45189889) == 'a MessageNotUnderstood occurred (error 2010), a String does not understand  #''abs'''
	-receiver oop(45211393) == '*'
	-selector oop(5408769) == #'abs'
	-tag oop(20) == nil

temporary variables and parameters:
----------------------------------------------------
[11] String (Object) >> doesNotUnderstand: @9 line 10
	receiver oop(45211393) == '*'

instance variables:

temporary variables and parameters:
	aMessageDescriptor oop(45211649) == abs
	args oop(45188609) == anArray( )
	ex oop(45192193) == a MessageNotUnderstood occurred (error 2010), a String does not understand  #'abs'
	sel oop(5408769) == #'abs'
----------------------------------------------------
[12] String (Object) >> _doesNotUnderstand:args:envId:reason: @12 line 17
	receiver oop(45211393) == '*'

instance variables:

temporary variables and parameters:
	aSmallInt oop(2) == 0
	aSymbol oop(5408769) == #'abs'
	anArray oop(45188609) == anArray( )
	dnuKind oop(2) == 0
----------------------------------------------------
[13] GsNMethod >> _breakOperation:forStepPoint:breakpointLevel: @2 line 9
	receiver oop(123967745) == GsNMethod OrderPricer>>numOfRegularDefaultPricedPlants

instance variables:
	-debugInfo oop(124667137) == anArray( 72057595916976128, 'numOfRegularDefaultPricedPlants
	"The count of plants that are not line item priced 
	and to which the normal schemes apply, that is, everything except basil"

	^(self ord...
	-iFields1 oop(144119586122377218) == 18014948265297152
	-inClass oop(124208641) == OrderPricer
	-selector oop(11915010) == 1489376

temporary variables and parameters:
	.debugInfo oop(20) == nil
	aStepPoint oop(45211393) == '*'
	anInt oop(2) == 0
	brkLevel oop(10) == 1
	info oop(20) == nil
----------------------------------------------------
[14] GsNMethod >> setBreakAtStepPoint:breakpointLevel: @3 line 12
	receiver oop(123967745) == GsNMethod OrderPricer>>numOfRegularDefaultPricedPlants

instance variables:
	-debugInfo oop(124667137) == anArray( 72057595916976128, 'numOfRegularDefaultPricedPlants
	"The count of plants that are not line item priced 
	and to which the normal schemes apply, that is, everything except basil"

	^(self ord...
	-iFields1 oop(144119586122377218) == 18014948265297152
	-inClass oop(124208641) == OrderPricer
	-selector oop(11915010) == 1489376

temporary variables and parameters:
	.debugInfo oop(20) == nil
	aStepPoint oop(45211393) == '*'
	brkLevel oop(10) == 1
----------------------------------------------------
[15] RowanMethodService >> setBreakAt: @8 line 6
	receiver oop(49702657) == aRowanMethodService(OrderPricer>>numOfRegularDefaultPricedPlants)

instance variables:
	-_connection oop(18532353) == aRsrConnection
	-_id oop(37882) == 4735
	-accessedInstVars oop(45214977) == anArray( )
	-breakPoints oop(45215489) == anOrderedCollection( )
	-category oop(45213441) == 'Support'
	-className oop(45214465) == 'OrderPricer'
	-command oop(76095489) == #'setBreakAt:'
	-commandArgs oop(45212673) == anArray( '*')
	-comparisonSource oop(20) == nil
	-compilationWarnings oop(20) == nil
	-definedClassName oop(45216001) == 'OrderPricer'
	-definedPackage oop(45215745) == 'Plantis'
	-failedCompile oop(12) == false
	-firstReference oop(20) == nil
	-hasMethodHistory oop(12) == false
	-hasSubs oop(12) == false
	-hasSupers oop(12) == false
	-homeMethodOop oop(991741962) == 123967745
	-inSelectedPackage oop(268) == true
	-isExtension oop(12) == false
	-isMethodForBlock oop(12) == false
	-isTestMethod oop(12) == false
	-meta oop(12) == false
	-methodDefinitions oop(20) == nil
	-oop oop(991741962) == 123967745
	-organizer oop(20) == nil
	-packageName oop(45213697) == 'Plantis'
	-projectName oop(45213953) == 'Plantis'
	-references oop(20) == nil
	-remoteSelf oop(45212417) == aRsrForwarder
	-renamedName oop(20) == nil
	-searchString oop(20) == nil
	-selectedPackageServices oop(20) == nil
	-selector oop(11915009) == #'numOfRegularDefaultPricedPlants'
	-shouldUpdate oop(268) == true
	-source oop(45213185) == 'numOfRegularDefaultPricedPlants
	"The count of plants that are not line item priced 
	and to which the normal schemes apply, that is, everything except basil"

	^(self order lineItems notLineItemPric...
	-stepPoints oop(45214721) == anArray( )
	-superDisplayString oop(20) == nil
	-testResult oop(20) == nil
	-testRunClassName oop(20) == nil
	-updateType oop(20) == nil
	-updates oop(45212929) == anArray( )
	-wasUpdated oop(268) == true

temporary variables and parameters:
	method oop(123967745) == GsNMethod OrderPricer>>numOfRegularDefaultPricedPlants
	stepPoint oop(45211393) == '*'
----------------------------------------------------
[16] RowanMethodService (Object) >> perform:withArguments: @1 line 12
	receiver oop(49702657) == aRowanMethodService(OrderPricer>>numOfRegularDefaultPricedPlants)

instance variables:
	-_connection oop(18532353) == aRsrConnection
	-_id oop(37882) == 4735
	-accessedInstVars oop(45214977) == anArray( )
	-breakPoints oop(45215489) == anOrderedCollection( )
	-category oop(45213441) == 'Support'
	-className oop(45214465) == 'OrderPricer'
	-command oop(76095489) == #'setBreakAt:'
	-commandArgs oop(45212673) == anArray( '*')
	-comparisonSource oop(20) == nil
	-compilationWarnings oop(20) == nil
	-definedClassName oop(45216001) == 'OrderPricer'
	-definedPackage oop(45215745) == 'Plantis'
	-failedCompile oop(12) == false
	-firstReference oop(20) == nil
	-hasMethodHistory oop(12) == false
	-hasSubs oop(12) == false
	-hasSupers oop(12) == false
	-homeMethodOop oop(991741962) == 123967745
	-inSelectedPackage oop(268) == true
	-isExtension oop(12) == false
	-isMethodForBlock oop(12) == false
	-isTestMethod oop(12) == false
	-meta oop(12) == false
	-methodDefinitions oop(20) == nil
	-oop oop(991741962) == 123967745
	-organizer oop(20) == nil
	-packageName oop(45213697) == 'Plantis'
	-projectName oop(45213953) == 'Plantis'
	-references oop(20) == nil
	-remoteSelf oop(45212417) == aRsrForwarder
	-renamedName oop(20) == nil
	-searchString oop(20) == nil
	-selectedPackageServices oop(20) == nil
	-selector oop(11915009) == #'numOfRegularDefaultPricedPlants'
	-shouldUpdate oop(268) == true
	-source oop(45213185) == 'numOfRegularDefaultPricedPlants
	"The count of plants that are not line item priced 
	and to which the normal schemes apply, that is, everything except basil"

	^(self order lineItems notLineItemPric...
	-stepPoints oop(45214721) == anArray( )
	-superDisplayString oop(20) == nil
	-testResult oop(20) == nil
	-testRunClassName oop(20) == nil
	-updateType oop(20) == nil
	-updates oop(45212929) == anArray( )
	-wasUpdated oop(268) == true

temporary variables and parameters:
	aSelectorSymbol oop(76095489) == #'setBreakAt:'
	anArray oop(45212673) == anArray( '*')
----------------------------------------------------
[17] RowanMethodService (RowanService) >> servicePerform:withArguments:shouldUpdate: @5 line 7
	receiver oop(49702657) == aRowanMethodService(OrderPricer>>numOfRegularDefaultPricedPlants)

instance variables:
	-_connection oop(18532353) == aRsrConnection
	-_id oop(37882) == 4735
	-accessedInstVars oop(45214977) == anArray( )
	-breakPoints oop(45215489) == anOrderedCollection( )
	-category oop(45213441) == 'Support'
	-className oop(45214465) == 'OrderPricer'
	-command oop(76095489) == #'setBreakAt:'
	-commandArgs oop(45212673) == anArray( '*')
	-comparisonSource oop(20) == nil
	-compilationWarnings oop(20) == nil
	-definedClassName oop(45216001) == 'OrderPricer'
	-definedPackage oop(45215745) == 'Plantis'
	-failedCompile oop(12) == false
	-firstReference oop(20) == nil
	-hasMethodHistory oop(12) == false
	-hasSubs oop(12) == false
	-hasSupers oop(12) == false
	-homeMethodOop oop(991741962) == 123967745
	-inSelectedPackage oop(268) == true
	-isExtension oop(12) == false
	-isMethodForBlock oop(12) == false
	-isTestMethod oop(12) == false
	-meta oop(12) == false
	-methodDefinitions oop(20) == nil
	-oop oop(991741962) == 123967745
	-organizer oop(20) == nil
	-packageName oop(45213697) == 'Plantis'
	-projectName oop(45213953) == 'Plantis'
	-references oop(20) == nil
	-remoteSelf oop(45212417) == aRsrForwarder
	-renamedName oop(20) == nil
	-searchString oop(20) == nil
	-selectedPackageServices oop(20) == nil
	-selector oop(11915009) == #'numOfRegularDefaultPricedPlants'
	-shouldUpdate oop(268) == true
	-source oop(45213185) == 'numOfRegularDefaultPricedPlants
	"The count of plants that are not line item priced 
	and to which the normal schemes apply, that is, everything except basil"

	^(self order lineItems notLineItemPric...
	-stepPoints oop(45214721) == anArray( )
	-superDisplayString oop(20) == nil
	-testResult oop(20) == nil
	-testRunClassName oop(20) == nil
	-updateType oop(20) == nil
	-updates oop(45212929) == anArray( )
	-wasUpdated oop(268) == true

temporary variables and parameters:
	collection oop(45212673) == anArray( '*')
	commandSymbol oop(76095489) == #'setBreakAt:'
	possiblyUpdate oop(268) == true
----------------------------------------------------
[18] RowanMethodService >> servicePerform:withArguments: @9 line 7
	receiver oop(49702657) == aRowanMethodService(OrderPricer>>numOfRegularDefaultPricedPlants)

instance variables:
	-_connection oop(18532353) == aRsrConnection
	-_id oop(37882) == 4735
	-accessedInstVars oop(45214977) == anArray( )
	-breakPoints oop(45215489) == anOrderedCollection( )
	-category oop(45213441) == 'Support'
	-className oop(45214465) == 'OrderPricer'
	-command oop(76095489) == #'setBreakAt:'
	-commandArgs oop(45212673) == anArray( '*')
	-comparisonSource oop(20) == nil
	-compilationWarnings oop(20) == nil
	-definedClassName oop(45216001) == 'OrderPricer'
	-definedPackage oop(45215745) == 'Plantis'
	-failedCompile oop(12) == false
	-firstReference oop(20) == nil
	-hasMethodHistory oop(12) == false
	-hasSubs oop(12) == false
	-hasSupers oop(12) == false
	-homeMethodOop oop(991741962) == 123967745
	-inSelectedPackage oop(268) == true
	-isExtension oop(12) == false
	-isMethodForBlock oop(12) == false
	-isTestMethod oop(12) == false
	-meta oop(12) == false
	-methodDefinitions oop(20) == nil
	-oop oop(991741962) == 123967745
	-organizer oop(20) == nil
	-packageName oop(45213697) == 'Plantis'
	-projectName oop(45213953) == 'Plantis'
	-references oop(20) == nil
	-remoteSelf oop(45212417) == aRsrForwarder
	-renamedName oop(20) == nil
	-searchString oop(20) == nil
	-selectedPackageServices oop(20) == nil
	-selector oop(11915009) == #'numOfRegularDefaultPricedPlants'
	-shouldUpdate oop(268) == true
	-source oop(45213185) == 'numOfRegularDefaultPricedPlants
	"The count of plants that are not line item priced 
	and to which the normal schemes apply, that is, everything except basil"

	^(self order lineItems notLineItemPric...
	-stepPoints oop(45214721) == anArray( )
	-superDisplayString oop(20) == nil
	-testResult oop(20) == nil
	-testRunClassName oop(20) == nil
	-updateType oop(20) == nil
	-updates oop(45212929) == anArray( )
	-wasUpdated oop(268) == true

temporary variables and parameters:
	collection oop(45212673) == anArray( '*')
	symbol oop(76095489) == #'setBreakAt:'
	theCommand oop(76095489) == #'setBreakAt:'
----------------------------------------------------
[19] [] in RowanService >> executeCommand @21 line 11
	receiver oop(45184257) == anExecBlock0

instance variables:

temporary variables and parameters:
	self oop(49702657) == aRowanMethodService(OrderPricer>>numOfRegularDefaultPricedPlants)
----------------------------------------------------
[20] ExecBlock0 (ExecBlock) >> on:do: @3 line 44
	receiver oop(45184257) == anExecBlock0

instance variables:

temporary variables and parameters:
	exceptionSelector oop(3540225) == GsInteractionRequest
	fastPath oop(268) == true
	handlerBlock oop(45215233) == anExecBlock
----------------------------------------------------
[21] [] in RowanService >> executeCommand @10 line 12
	receiver oop(45216257) == anExecBlock0

instance variables:

temporary variables and parameters:
	self oop(49702657) == aRowanMethodService(OrderPricer>>numOfRegularDefaultPricedPlants)
----------------------------------------------------
[22] ExecBlock0 (ExecBlock) >> on:do: @3 line 44
	receiver oop(45216257) == anExecBlock0

instance variables:

temporary variables and parameters:
	exceptionSelector oop(137729) == Exception
	fastPath oop(268) == true
	handlerBlock oop(79628545) == anExecBlock1
----------------------------------------------------
[23] RowanMethodService (RowanService) >> executeCommand @4 line 19
	receiver oop(49702657) == aRowanMethodService(OrderPricer>>numOfRegularDefaultPricedPlants)

instance variables:
	-_connection oop(18532353) == aRsrConnection
	-_id oop(37882) == 4735
	-accessedInstVars oop(45214977) == anArray( )
	-breakPoints oop(45215489) == anOrderedCollection( )
	-category oop(45213441) == 'Support'
	-className oop(45214465) == 'OrderPricer'
	-command oop(76095489) == #'setBreakAt:'
	-commandArgs oop(45212673) == anArray( '*')
	-comparisonSource oop(20) == nil
	-compilationWarnings oop(20) == nil
	-definedClassName oop(45216001) == 'OrderPricer'
	-definedPackage oop(45215745) == 'Plantis'
	-failedCompile oop(12) == false
	-firstReference oop(20) == nil
	-hasMethodHistory oop(12) == false
	-hasSubs oop(12) == false
	-hasSupers oop(12) == false
	-homeMethodOop oop(991741962) == 123967745
	-inSelectedPackage oop(268) == true
	-isExtension oop(12) == false
	-isMethodForBlock oop(12) == false
	-isTestMethod oop(12) == false
	-meta oop(12) == false
	-methodDefinitions oop(20) == nil
	-oop oop(991741962) == 123967745
	-organizer oop(20) == nil
	-packageName oop(45213697) == 'Plantis'
	-projectName oop(45213953) == 'Plantis'
	-references oop(20) == nil
	-remoteSelf oop(45212417) == aRsrForwarder
	-renamedName oop(20) == nil
	-searchString oop(20) == nil
	-selectedPackageServices oop(20) == nil
	-selector oop(11915009) == #'numOfRegularDefaultPricedPlants'
	-shouldUpdate oop(268) == true
	-source oop(45213185) == 'numOfRegularDefaultPricedPlants
	"The count of plants that are not line item priced 
	and to which the normal schemes apply, that is, everything except basil"

	^(self order lineItems notLineItemPric...
	-stepPoints oop(45214721) == anArray( )
	-superDisplayString oop(20) == nil
	-testResult oop(20) == nil
	-testRunClassName oop(20) == nil
	-updateType oop(20) == nil
	-updates oop(45212929) == anArray( )
	-wasUpdated oop(268) == true

temporary variables and parameters:
----------------------------------------------------
[24] RowanMethodService (Object) >> perform:withArguments: @1 line 12
	receiver oop(49702657) == aRowanMethodService(OrderPricer>>numOfRegularDefaultPricedPlants)

instance variables:
	-_connection oop(18532353) == aRsrConnection
	-_id oop(37882) == 4735
	-accessedInstVars oop(45214977) == anArray( )
	-breakPoints oop(45215489) == anOrderedCollection( )
	-category oop(45213441) == 'Support'
	-className oop(45214465) == 'OrderPricer'
	-command oop(76095489) == #'setBreakAt:'
	-commandArgs oop(45212673) == anArray( '*')
	-comparisonSource oop(20) == nil
	-compilationWarnings oop(20) == nil
	-definedClassName oop(45216001) == 'OrderPricer'
	-definedPackage oop(45215745) == 'Plantis'
	-failedCompile oop(12) == false
	-firstReference oop(20) == nil
	-hasMethodHistory oop(12) == false
	-hasSubs oop(12) == false
	-hasSupers oop(12) == false
	-homeMethodOop oop(991741962) == 123967745
	-inSelectedPackage oop(268) == true
	-isExtension oop(12) == false
	-isMethodForBlock oop(12) == false
	-isTestMethod oop(12) == false
	-meta oop(12) == false
	-methodDefinitions oop(20) == nil
	-oop oop(991741962) == 123967745
	-organizer oop(20) == nil
	-packageName oop(45213697) == 'Plantis'
	-projectName oop(45213953) == 'Plantis'
	-references oop(20) == nil
	-remoteSelf oop(45212417) == aRsrForwarder
	-renamedName oop(20) == nil
	-searchString oop(20) == nil
	-selectedPackageServices oop(20) == nil
	-selector oop(11915009) == #'numOfRegularDefaultPricedPlants'
	-shouldUpdate oop(268) == true
	-source oop(45213185) == 'numOfRegularDefaultPricedPlants
	"The count of plants that are not line item priced 
	and to which the normal schemes apply, that is, everything except basil"

	^(self order lineItems notLineItemPric...
	-stepPoints oop(45214721) == anArray( )
	-superDisplayString oop(20) == nil
	-testResult oop(20) == nil
	-testRunClassName oop(20) == nil
	-updateType oop(20) == nil
	-updates oop(45212929) == anArray( )
	-wasUpdated oop(268) == true

temporary variables and parameters:
	aSelectorSymbol oop(76184321) == #'executeCommand'
	anArray oop(45186561) == anArray( )
----------------------------------------------------
[25] RsrMessageSend >> perform @5 line 4
	receiver oop(45181185) == aRsrMessageSend

instance variables:
	-arguments oop(45186561) == anArray( )
	-receiver oop(49702657) == aRowanMethodService(OrderPricer>>numOfRegularDefaultPricedPlants)
	-selector oop(76184321) == #'executeCommand'

temporary variables and parameters:
----------------------------------------------------
[26] [] in RsrSendMessage >> perform:answerUsing: @9 line 6
	receiver oop(45192705) == anExecBlock0

instance variables:

temporary variables and parameters:
	aMessageSend oop(45181185) == aRsrMessageSend
	aResolver oop(45192961) == aRsrRemotePromiseResolver
	result oop(20) == nil
----------------------------------------------------
[27] ExecBlock0 (ExecBlock) >> on:do: @3 line 44
	receiver oop(45192705) == anExecBlock0

instance variables:

temporary variables and parameters:
	exceptionSelector oop(77249537) == RsrUnhandledException
	fastPath oop(268) == true
	handlerBlock oop(45193217) == anExecBlock
----------------------------------------------------
[28] RsrSendMessage >> perform:answerUsing: @3 line 9
	receiver oop(45192449) == aRsrSendMessage

instance variables:
	-argumentReferences oop(45217793) == anArray( )
	-receiverReference oop(45220609) == aRsrServiceReference
	-selectorReference oop(45218049) == aRsrSymbolReference
	-snapshots oop(45193473) == anArray( aRsrServiceSnapshot)
	-transaction oop(18446744073709550914) == -88

temporary variables and parameters:
	aMessageSend oop(45181185) == aRsrMessageSend
	aResolver oop(45192961) == aRsrRemotePromiseResolver
----------------------------------------------------
[29] [] in RsrSendMessage >> executeFor: @34 line 21
	receiver oop(45217537) == anExecBlock0

instance variables:

temporary variables and parameters:
	aConnection oop(18532353) == aRsrConnection
	arguments oop(45186561) == anArray( )
	messageSend oop(45181185) == aRsrMessageSend
	receiver oop(49702657) == aRowanMethodService(OrderPricer>>numOfRegularDefaultPricedPlants)
	resolver oop(45192961) == aRsrRemotePromiseResolver
	selector oop(76184321) == #'executeCommand'
	self oop(45192449) == aRsrSendMessage
	servicesStrongly oop(20) == nil
----------------------------------------------------
[30] ExecBlock0 (ExecBlock) >> on:do: @3 line 44
	receiver oop(45217537) == anExecBlock0

instance variables:

temporary variables and parameters:
	exceptionSelector oop(77249537) == RsrUnhandledException
	fastPath oop(268) == true
	handlerBlock oop(45217281) == anExecBlock
----------------------------------------------------
[31] [] in RsrSendMessage >> executeFor: @9 line 22
	receiver oop(45217025) == anExecBlock0

instance variables:

temporary variables and parameters:
	self oop(45192449) == aRsrSendMessage
----------------------------------------------------
[32] ExecBlock0 (ExecBlock) >> ensure: @2 line 12
	receiver oop(45217025) == anExecBlock0

instance variables:

temporary variables and parameters:
	b oop(20) == nil
	result oop(20) == nil
	terminationBlock oop(45216769) == anExecBlock
----------------------------------------------------
[33] RsrSendMessage >> executeFor: @4 line 26
	receiver oop(45192449) == aRsrSendMessage

instance variables:
	-argumentReferences oop(45217793) == anArray( )
	-receiverReference oop(45220609) == aRsrServiceReference
	-selectorReference oop(45218049) == aRsrSymbolReference
	-snapshots oop(45193473) == anArray( aRsrServiceSnapshot)
	-transaction oop(18446744073709550914) == -88

temporary variables and parameters:
	aConnection oop(18532353) == aRsrConnection
	arguments oop(45186561) == anArray( )
	messageSend oop(45181185) == aRsrMessageSend
	receiver oop(49702657) == aRowanMethodService(OrderPricer>>numOfRegularDefaultPricedPlants)
	resolver oop(45192961) == aRsrRemotePromiseResolver
	selector oop(76184321) == #'executeCommand'
	servicesStrongly oop(20) == nil
----------------------------------------------------
[34] [] in RsrConnection >> _receivedCommand: @10 line 7
	receiver oop(45216513) == anExecBlock0

instance variables:

temporary variables and parameters:
	aCommand oop(45192449) == aRsrSendMessage
	self oop(18532353) == aRsrConnection
----------------------------------------------------
[35] [] in RsrProcessModel >> fork:named: @7 line 5
	receiver oop(45190657) == anExecBlock

instance variables:

temporary variables and parameters:
	aBlock oop(45216513) == anExecBlock0
	aString oop(45214209) == 'Processing RsrSendMessage'
	self oop(18499329) == aRsrProcessModel
----------------------------------------------------
[36] ExecBlock >> valueWithArguments: @1 line 7
	receiver oop(45190657) == anExecBlock

instance variables:

temporary variables and parameters:
	argList oop(233217) == anArray( )
----------------------------------------------------
[37] GsProcess >> _start @7 line 16
	receiver oop(45212161) == GsProcess(oop=45212161, status=suspended, priority=15, block=anExecBlock, args=anArray( ), 
   ProcessorScheduler >> _switchFrom:to: @2 line 8
   ProcessorScheduler >> _reschedule @7 line 7
   Process...

instance variables:
	-arStack oop(45190401) == aGsStackBuffer( 64, GsNMethod ProcessorScheduler>>_switchFrom:to:, -1888, 136, GsProcess(oop=45191169, status=active, priority=15, block=anExecBlock, args=anArray( ), 
   GsProcess >> _reportAt: @7 li...
	-args oop(233217) == anArray( )
	-block oop(45190657) == anExecBlock
	-blockResult oop(276) == remoteNil
	-breakpointLevel oop(10) == 1
	-clientData oop(20) == nil
	-dbgFpsCache oop(45190913) == anArray( 3, 11, 17, 22, 29, 34, 41, 46, 51, 55, 62, 67, 77, 84, 91, 96, 102, 110, 117, 124, 131, 138, 147, 151, 157, 163, 170, 177, 184, 191, 198, 206, 214, 220, 225, 229, 236, 244)
	-debugActionBlock oop(45191425) == anExecBlock1
	-environment oop(20) == nil
	-ffiErrno oop(2) == 0
	-group oop(20) == nil
	-interruptFlag_stEntryCount oop(10) == 1
	-ioSelectResult oop(20) == nil
	-joiners oop(20) == nil
	-lastBreakpt oop(20) == nil
	-modeInfo oop(86033565698) == 10754195712
	-msgEnvironmentId oop(2) == 0
	-name oop(45191681) == 'RowanMethodService>>executeCommand'
	-onQueue oop(18975745) == anIdentitySet( GsProcess(oop=26702849, status=suspended, priority=15, block=anExecBlock, args=anArray( ), 
   ProcessorScheduler >> _switchFrom:to: @2 line 8
   ProcessorScheduler >> _reschedule @7 li...
	-parentProcess oop(18366977) == GsProcess(oop=18366977, status=waiting on a GsSignalingSocket, priority=30, block=anExecBlock, args=anArray( ), 
   ProcessorScheduler >> _switchFrom:to: @2 line 8
   ProcessorScheduler >> _reschedule...
	-priority_use oop(131941395333162) == 16492674416645
	-signalTime oop(20) == nil
	-signalledException oop(20) == nil
	-stackId oop(18446744073709551610) == -1
	-stackSerialNum oop(62013669418) == 7751708677
	-startSeconds oop(11269708529815813222) == 750880345.2326591
	-topFpOffset oop(18446744073709536002) == -1952
	-uaCount_depth oop(306) == 38
	-waitingOn oop(20) == nil

temporary variables and parameters:
	curr oop(20) == nil
	res oop(20) == nil
----------------------------------------------------
[38] GsNMethod class >> _gsReturnToC @1 line 11
	receiver oop(144897) == GsNMethod

instance variables:

temporary variables and parameters:
----------------------------------------------------
```

**Comment by ericwinger** [comment]
Make sure breakpoint menu option isn't available if not on a step point and double check that we are actually sending a step point to the server when setting breakpoint.

**Comment by LisaAlmarode** [comment]
No walkback, but it's quite annoying and difficult to set a breakpoint.  I will file a separate issue.
